Libraries/Microsoft.Teams.Apps/App.cs

```diff
@@ -250,6 +252,19 @@ public async Task<Response> Process(ISenderPlugin sender, IToken token, IActivit
             return res;
         }
 
+        // merge extra data into context extras
+        var mergedExtra = new Dictionary<string, object>(ContextExtra);
+        foreach (var plugin in Plugins)
+        {
+            if (plugin.ContextExtra is not null)
+            {
+                foreach (var kvp in plugin.ContextExtra)
+                {
+                    mergedExtra[kvp.Key] = kvp.Value;
```

Comment:
if there was already an entry, this would override right? should we add that as a note somewhere?

Reply:
once this PR gets merged i'll write some docs for this feature (possibly in [this](https://microsoft.github.io/teams-ai/typescript/essentials/app-basics#plugins) section) - as for this i'll add a comment